### PR TITLE
add Jeremie to code owners

### PR DESCRIPTION
**Pull request by adrinjalali**
@jeremiedbb managed the latest release. Adding him to code owners here.

Ping @ogrisel

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -90,3 +90,4 @@ extra:
     - rth
     - adrinjalali
     - glemaitre
+    - jeremiedbb
```
Comment (jakirkham) [inline recipe/meta.yaml]: FWIW only this change is the only one that needs to be done manually. Re-rendering will pick up the rest 🙂

Comment (adrinjalali) [reply recipe/meta.yaml]: thanks @jakirkham , good to know :)

**Comment by rth** [comment]
Sorry for merging earlier, I though those other changes were done by a bot )

**Comment by adrinjalali** [comment]
@rth, I'm happy I can pass as a bot lolol

**Pull request merged**